PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/SparkLivySessionOutputStream.kt

```diff
@@ -41,13 +41,15 @@ class SparkLivySessionOutputStream(val session: Session) : ByteArrayOutputStream
 
         session.runCodes(codes)
                 .doOnEach { reset() }
+                .toBlocking()
```

Comment:
Will it block UI thread if we make this thread blocking

Reply:
Yes, it will block the current thread a little ... And the impact is bigger than I imaged. Thanks!